games/space-invaders/Space Invaders [David Winter].json

```diff
@@ -0,0 +1,108 @@
+{
+  "title": "Space Invaders",
+  "authors": [
+    {
+      "id": "david-winter",
+      "name": "David Winter"
+    }
+  ],
+  "release": "1978",
+  "category": "GAME",
+  "description": "The well known game. Destroy the invaders with your ship.",
+  "images": [
+    "./Space Invaders [David Winter] - main screen.png",
+    "./Space Invaders [David Winter] - game play.png"
+  ],
+  "interpreter": {
+    "id": "chip8-david-winter",
+    "name": "CHIP8 by David Winter"
+  },
+  "binaries": {
+    "names": [
+      { "value": "Space Invaders [David Winter] - 0.9.1" },
+      { "value": "Space Invaders [David Winter]" },
+
+      { "value": "Space Invaders [David Winter] - 0.9.0" },
+      { "value": "Space Invaders [David Winter] (alt)" },
+      { "value": "INVADERS" }
+    ],
```

Comment:
`binaries.names` is a list of commonly used names (without extensions) of ROM files. I also included versions (since they are embedded in binaries). I'm thinking if it would be feasible to create separate `.json` files per version. Some information would be duplicated. I guess the decision would have to be made on per binary basis (I think x.x.1 here means it's a bugfix so it would be an overkill)

This can be used in emulator as a fallback lookup when ROM with unknown hash is selected.

Reply:
Good idea! Would probably be more succint to have a JSON array instead of an object where every key is "value" though.

Reply:
This is futureproofing. If we want to add something more than `value` in the future we will have a way (like `version`).

Reply:
I reorganized binaries objects by aggregating information in names and hashes. Looks quite nice.